### PR TITLE
Add Resource Allocation

### PR DESCRIPTION
**Pull request by tuannvm**
## Why?

- Add the config for resource limit / request

### CODE REVIEW

File: deployments/helm/signalfx-k8s-metrics-adapter/values.yaml [hunk 1]
```diff
@@ -9,3 +9,10 @@ imageTag: 0.2.0
 imagePullPolicy: IfNotPresent
 args:
 
+resources:
```
Comment (keitwb) [inline deployments/helm/signalfx-k8s-metrics-adapter/values.yaml]: It is better to not provide any defaults for resources and just let it continue without resource bounds by default.

Comment (tommynguyen-vungle) [reply deployments/helm/signalfx-k8s-metrics-adapter/values.yaml]: @keitwb comment addressed, please help check, thanks.

**Review by keitwb — APPROVED** [review]
Looks good, thanks for adding this.

**Comment by keitwb** [comment]
Ok, I released version 0.0.3 to the helm repo.

**Pull request merged**